### PR TITLE
Wrap link teaser article

### LINKED ISSUE #1071: Wrap news article teaser in anchor tag
At the moment news article teasers (as can be found on the news overview landing page and in the related news article section at the bottom of a news article) are not clickable as a whole. Only the title of the article is clickable.
This is also a problem for how we want to track links through GTM. 

- make the whole article teaser 'card' link through to the article.

screen shot 2017-07-18 at 16 30 42

### PR DESCRIPTION
**Pull request by gusliedke**
fixes #1071 

https://github.com/comicrelief/pattern-lab/pull/194

### CODE REVIEW

File: themes/custom/campaign_base/package.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "name": "Campaign_base",
   "version": "1.0.0",
   "dependencies": {
-    "@comicrelief/pattern-lab": "^5.1.7",
+    "@comicrelief/pattern-lab": "git://github.com/comicrelief/pattern-lab.git#e183fd0",
```
Comment (AndyEPhipps) [inline themes/custom/campaign_base/package.json]: Guess we want to revert this before it goes merged on?

Comment (gusliedke) [inline themes/custom/campaign_base/package.json]: oh yes

Comment (gusliedke) [inline themes/custom/campaign_base/package.json]: Can you merge it first, please 1071_wrap_link_teaser_article to get right version

Comment (AndyEPhipps) [inline themes/custom/campaign_base/package.json]: Have merged that PR!

**Comment by Saphyel** [comment]
@gusliedke we are not using a lock file yet, right?

**Comment by gusliedke** [comment]
@Saphyel no, lock file is a npm 5 thing. maybe we can move to yarn some time soon

**Pull request merged**